examples/juniper/ncptx/ncptx-cptx.pb.txt

```diff
@@ -0,0 +1,154 @@
+name: "ncptx-cptx-nsoni"
+nodes: {
+    name: "ncptx"
+    vendor: JUNIPER
+    model: "ncptx"
+    os: "evo"
+    config: {
+      config_path: "/home/evo/configdisk/"
+      config_file: "juniper.conf"
+      file: "juniper.config"
+      image: "ncptx:latest"
```

Comment:
likewise with the image field, this should be added to the node defaults if model is ncptx

Reply:
Done. Removed other default services as well.